ds3-sdk/src/main/java/com/spectralogic/ds3client/serializer/XmlOutput.java

```diff
@@ -92,8 +92,8 @@ private static String toXml(final Object object, final FilterProvider filterProv
         }
     }
 
-    public static String toXml(final Ds3ObjectList objects, final BulkCommand command) {
-        if (command == BulkCommand.GET) {
```

Comment:
This was written before we had the other calls that did not need the size param.  So to support the new api calls (get physical placement, verify physical placement, create verify job, etc) we need to invert this logic to only look for bulk PUT.